samples/bluetooth/throughput/README.rst

```diff
@@ -55,15 +55,21 @@ By default, the following connection parameter values are used:
 Changing connection parameter values
 ====================================
 
-You can experiment with different connection parameter values by reconfiguring the values using menuconfig and then compiling and programming the sample again to at least one of the boards.
+You can experiment with different connection parameter values by reconfiguring the values using the :ref:`zephyr:shell_api` interface before running a test.
+The following parameters can be adjusted:
```

Comment:
You can adjust the following parameters:

Reply:
Done